pkg/services/ngalert/api/tooling/definitions/contact_points.go

```diff
@@ -188,8 +188,9 @@ type SlackIntegration struct {
 type TelegramIntegration struct {
 	DisableResolveMessage *bool `json:"-" yaml:"-" hcl:"disable_resolve_message"`
 
-	BotToken Secret `json:"bottoken" yaml:"bottoken" hcl:"token"`
-	ChatID   string `json:"chatid,omitempty" yaml:"chatid,omitempty" hcl:"chat_id"`
+	BotToken        Secret `json:"bottoken" yaml:"bottoken" hcl:"token"`
+	ChatID          string `json:"chatid,omitempty" yaml:"chatid,omitempty" hcl:"chat_id"`
+	MessageThreadID string `json:"messagethreadid,omitempty" yaml:"messagethreadid,omitempty" hcl:"message_thread_id"`
```

Comment:
please use correct tags for JSON and YAML, i.e `message_thread_id`

Reply:
Done.